pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -413,3 +400,16 @@ def teardown(self) -> None:
             self.lightning_module.cpu()
             # clean up memory
             torch.cuda.empty_cache()
+
+
+class _ExtraQueue(list):
+    """Simulates a :class:`torch.multiprocessing.queue.SimpleQueue` interface using the Python list."""
+
+    def get(self) -> Any:
+        return self.pop(0)
+
+    def put(self, item: Any) -> None:
+        self.append(item)
+
+    def empty(self) -> bool:
+        return len(self) == 0
```

Comment:
Reviewer: This is needed to preserve backward compatibility with everyone overriding the `get_form_queue` and `add_to_queue` hooks expecting a queue object. See #10865 for how I propose to go forward.